core/domain/summary_services.py

```diff
@@ -23,6 +23,17 @@
 import utils
 
 _LIBRARY_INDEX_GROUPS = [{
+    'header': 'Recently Published',
+    'search_categories': [
```

Comment:
I'd suggest it might be better if you created a new function that gathers the `Recently Published` group rather than via searching for it. The main reason is that the `Recently Published` group is supposed to capture everything that is non-private (even categories that you don't have listed here).